### ISSUE TITLE
Error Message From todo.py on Right Click on Tree Node

### ISSUE THREAD

**Issue by tbpassin** [leo-editor/leo-editor#2856]
The expected context menu came up despite this message.  One mystery - the todo plugin is not one of my enabled plugins but it's been showing up lately anyway.  I don't really like that.

```
Exception executing right-click handler
Traceback (most recent call last):

  File "c:\Tom\git\leo-editor\leo\plugins\qt_gui.py", line 1193, in onContextMenu
    handler(c, p, menu)

  File "c:\Tom\git\leo-editor\leo\plugins\todo.py", line 124, in popup_entry
    c.cleo.addPopupMenu(c, p, menu)

AttributeError: 'Commands' object has no attribute 'cleo'
```

```
Leo 6.7.0-devel, devel branch, build 88e8a60a8e
2022-09-14 06:46:32 -0500
Python 3.10.4, PyQt version 6.3.0
Windows 10 AMD64 (build 10.0.19043) SP0
```

**Comment by tbpassin** [comment]
Ah, I see.  The todo plugin has been enabled in leoSettings.leo.  Can we please revert this change?  I don't understand its display, it clutters up my log pane toolbar, and I doubt I'll every use it.  It should be an option in myLeoSettings.leo, not forced on for everyone.

**Comment by edreamleo** [comment]
@tbpassin Unlike most settings, the `@enabled-plugins` node in myLeoSettings.leo (or any other file) *replaces* the same node in leoSettings.leo.  So 'todo.py' isn't "forced" on anyone.

**Comment by tbpassin** [comment]
That may have been the plan but it's commented out in my own myLeoSettings.leo, and I'm still seeing the plugin.

**Comment by tbpassin** [comment]
Oh, I will correct myself - I only see the todo plugin with LeoPyRef.

**Comment by edreamleo** [comment]
@tbpassin Thanks for the clarification. I double-checked how Leo works before writing my comment about `enabled-plugins` nodes. If you don't like the todo plugin you can disable it in your personal copy of LeoPyRef.leo, by convention called just leoPy.leo.

This leaves us with the initial bug report :-) I suspect the code fails if todo.py is enabled in some commanders but not others.  I'm on it...

**Comment by tbpassin** [comment]
As a quirky complication, I just opened an outline I haven't opened for a week or two, and there it is again - the Tasks tab.  This outline does have an @settings node but no settings involving the todo plugin.

**Comment by edreamleo** [comment]
Fixed via PR #2860.
